gui/hud.gui

```diff
@@ -3552,6 +3727,388 @@ types HUD
+								texture = "gfx/interface/colors/dark_purple.dds"
+							}
+						}
+						
+						progressbar_hud_infusion = {
+							visible = "[EqualTo_int32(FixedPointToInt(GetPlayer.MakeScope.Var('light_void_infusion_level').GetValue), '(int32)-1')]"
+							
+							blockoverride "bar_color"
+							{
+								texture = "gfx/interface/colors/purple.dds"
+							}
+						}
+						
+						progressbar_hud_infusion = {
+							visible = "[And(EqualTo_int32(FixedPointToInt(GetPlayer.MakeScope.Var('light_void_infusion_level').GetValue), '(int32)0'), LessThanOrEqualTo_int32(FixedPointToInt(GetPlayer.MakeScope.Var('light_void_infusion').GetValue), '(int32)-1'))]"
+							
+							blockoverride "bar_color"
+							{
+								texture = "gfx/interface/colors/bright_purple.dds"
+							}
+						}
+						
+						progressbar_hud_infusion = {
+							visible = "[And(EqualTo_int32(FixedPointToInt(GetPlayer.MakeScope.Var('light_void_infusion_level').GetValue), '(int32)0'), GreaterThanOrEqualTo_int32(FixedPointToInt(GetPlayer.MakeScope.Var('light_void_infusion').GetValue), '(int32)1'))]"
+							
+							blockoverride "bar_color"
+							{
+								texture = "gfx/interface/colors/gold_darker.dds"
+							}
+						}
+					}
+
+					widget = {
+						layoutpolicy_horizontal = expanding
+						layoutpolicy_vertical = expanding
+						block "stretch_2"
+						{
+							layoutstretchfactor_horizontal = 0
+						}
+					}
+				}
+			}
+
+			widget = {
+				size = { 100% 100% }
+
+				progressbar_hud_stress_glow = {
+					texture = "gfx/interface/hud/hud_infusion_glow_gold.dds"
+
+					blockoverride "animation_trigger_1"
+					{
+						name = "infusion_level_increase"
+					}
+
+					blockoverride "animation_trigger_2"
+					{
+						name = "stress_increase"
+					}
+				}
+
+				progressbar_hud_stress_glow = {
+					texture = "gfx/interface/hud/hud_infusion_glow_blue.dds"
+
+					blockoverride "animation_trigger_1"
+					{
+						name = "stress_level_decrease"
+					}
+
+					blockoverride "animation_trigger_2"
+					{
+						name = "stress_decrease"
+					}
+				}
+
+				progressbar_hud_stress_glow = {
+					visible = "[GreaterThanOrEqualTo_int32(FixedPointToInt(GetPlayer.MakeScope.Var('light_void_infusion_level').GetValue), '(int32)3')]"
+					texture = "gfx/interface/hud/hud_infusion_glow_gold.dds"
+
+					blockoverride "animation_trigger_1"
+					{
+						name = "lvl3_fire"
+						trigger_on_create = yes
+					}
+
+					blockoverride "animation_trigger_2" {}
+					blockoverride "lvl_3" {}
+				}
+
+				progressbar_hud_stress_glow = {
+					visible = "[LessThanOrEqualTo_int32(FixedPointToInt(GetPlayer.MakeScope.Var('light_void_infusion_level').GetValue), '(int32)-3')]"
+					texture = "gfx/interface/hud/hud_infusion_glow_purple.dds"
+
+					blockoverride "animation_trigger_1"
+					{
+						name = "lvl3_fire"
+						trigger_on_create = yes
+					}
+
+					blockoverride "animation_trigger_2" {}
+					blockoverride "lvl_3" {}
+				}
+
+				infusion_animation = {}
```

Comment:
`infusion_animation` is a template.
```suggestion
				using = infusion_animation
```
Also, words in template names are written with capital letters.